Documentation/ApiOverview/TypoScriptSyntax/Syntax/Conditions.rst

```diff
@@ -274,12 +279,15 @@ from the top level. This is normally done when TypoScript code from
 various records is combined.
 
 
+.. index:: TypoScript; Symfony expression language
 .. _typoscript-syntax-conditions-expression-language:
 
-Custom Conditions With Symfony Expression Language
-==================================================
+Custom conditions with the Symfony expression language
+======================================================
+
+Further information about how to extend TypoScript with your own custom
+conditions can be found within :ref:`sel-within-typoscript-conditions`.
```

Comment:
This would lead to "... within _Within TypoScript conditions_" since the title from the link target is used, right?

Reply:
yeah

Reply:
So we want to keep it that way?

Reply:
Better write: :ref:`TypoScript conditions <sel-within-typoscript-conditions>`

Reply:
ok, did so, ok now?